src/cli/managers/config_manager.py

```diff
@@ -1,16 +1,47 @@
+import os
 import logging
 from pathlib import Path
-from typing import Any, Dict, List, Tuple
-
-from cli.utils import read_json_file
+from typing import Dict, List, Tuple, Any
+from cli.utils import read_json_file, write_json_file
 
 
 class ConfigManager:
     def __init__(self, config_dir: Path):
         self.__config_dir = config_dir
+        self.app_config_file = self.__config_dir / "applications.json"
 
     def add_application(self, application: str, path: Path) -> None:
-        pass
+        """
+        Takes care of saving the application name in the .qne/application.json root file together with the application
+        path.
+
+        Args:
+            application: the application name
+            roles: a list of roles
+            path: the path where the application is stored
+
+        """
+
+        # Read json file
+        apps = read_json_file(self.app_config_file)
+
+        # Store the app path
+        apps[application] = {'path': os.path.join(str(path), application) + "/"}
```

Comment:
On Windows the trailing slash is inconsistent. To add a trailing slash OS independent, add join with ''
```suggestion
        apps[application] = {'path': os.path.join(str(path), application, '')}
```

Reply:
Now you do a nested join which is not needed, see my suggestion